### ISSUE TITLE
How can I solve the cuda capability problem?

### ISSUE THREAD

**Issue by feiji110** [txie-93/cdvae#34]
When I command 'python cdvae/run.py data=perov expname=perov',I meet the problem that CUDA capabilities probelm.The concrete hint is as follow:
```
packages/torch/cuda/__init__.py:104: UserWarning: 
NVIDIA GeForce RTX 4090 with CUDA capability sm_89 is not compatible with the current PyTorch installation.
The current PyTorch install supports CUDA capabilities sm_37 sm_50 sm_60 sm_61 sm_70 sm_75 compute_37.
```

**Comment by kyonofx** [comment]
Hi, looks like you could try upgrading your torch version to the latest version (1.13); it should not break anything.

**Comment by feiji110** [comment]
> Hi, looks like you could try upgrading your torch version to the latest version (1.13); it should not break anything.

Thank you! I'm glad to hear from you although I failed after I did as you say.After a night and a day I finally use pip to upgrade some paggages and solve this problem.Now I meet the probelm about ReduceLROnPlateau.The hint is as follow:
````py
 raise MisconfigurationException(
lightning_lite.utilities.exceptions.MisconfigurationException: ReduceLROnPlateau conditioned on metric val_loss which is not available.
```` 
Looking forward to your reply.

**Comment by grzegorzkaszuba** [comment]
Hi,
I'm currently working my way through the project to understand its components, and I have encountered the exact same problem.

Apparently, the method which adjusts the learning rates based on the current progress is called at the end of the training step - during the first epoch, no val_loss has been produced yet (perhaps adjustments could be made to do it after the validation step or omit the scheduler calls in the first epoch).  To simply get the training to run, you can modify the configurations. What worked for me was changing "monitor" to "train_loss" in model.py: BaseModule class.
